templates/index.html

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta name="google-site-verification" content="VBSIFp1OyI1gpnRTREBEjNBj_Wat50wzwzVTrG1YuHY" />
+    <meta name=“google-site-verification” content=“7nOT58tx-TbWUt6dokG5Yj1Chca3j5-qX5qQ4dTGVew” />
```

Comment:
@omobosteven, please correct the quotation marks so we steer clear of character encoding issues.

Reply:
Okay

Reply:
@joshuaocero I have made the correction.